### DISCUSSION TITLE
Got an unclear warning running a graph

### DISCUSSION THREAD

**Discussion by yuval-airis** [dagster-io/dagster#18298] [Q&A] [answered]
It said that 
UserWarning: While in @graph context X, received an uninvoked op Y

I called op Y inside the function of graph X, why is that an issue?
And how may I perform the calls to ops from within a graph? (I can't immediately say what's the difference between what I did and [this example](https://docs.dagster.io/concepts/ops-jobs-graphs/graphs#using-the-graph-decorator))

**Comment by smackesey** [comment, marked as answer]
Hi yuval, most likely you did this:

```
@graph
def my_graph():
   z(y)
```

It needs to be `z(y())`. Calling an op in a graph context is invoking it.

**Reply by yuval-airis** [reply]
Exactly the case. Thanks!
